ftp.go

```diff
@@ -535,6 +535,28 @@ func (c *ServerConn) List(path string) (entries []*Entry, err error) {
 	}
 	return
 }
+//Stat a MLST FTP command
+func (c *ServerConn) Stat(path string)(entry *Entry, err error){
+	var cmd string
+	var parser parseFunc
+
+	if c.mlstSupported {
+		cmd = "MLST"
+		parser = parseRFC3659ListLine
+	} else {
+		return nil, errors.New("unexpected MLST response")
+	}
+	_, msg, err := c.cmd(StatusRequestedFileActionOK, "%s %s", cmd, path)
+	if err!= nil{
+		return nil, err
+	}
+	lines := strings.Split(msg, "\n")
```

Comment:
What happens if the filename contains spaces?

Reply:
it's fine work contains spaces.